src/convertToObject.js

```diff
@@ -6,7 +6,26 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styleObject = {};
+  const cleanStrings = sourceString
+    .trim()
+    .split(';')
+    .filter((style) => style.length);
+
+  cleanStrings.forEach((cleanString) => {
+    const partsOfString = cleanString
+      .split(':')
+      .map((partStyle) => partStyle.trim());
+
+    const key = partsOfString[0];
+    const value = partsOfString[1];
+
+    if (key && value) {
```

Comment:
Consider adding a check to ensure that `partsOfString` has exactly two elements before assigning `key` and `value`. This will prevent potential errors if the input string contains malformed style declarations.

Reply:
if (partsOfString.length === 2) {...}

Reply:
@OkMoroz 
It sounds like a check to avoid something that will never happen to u)
it's redundant)